src/elm/Ui/Message.elm

```diff
@@ -73,6 +73,9 @@ statusToAttribute status =
         Warning _ ->
             Just <| A.attribute "aria-live" "polite"
 
+        Valid _ ->
+            Just <| A.attribute "aria-live" "polite"
+
         Error _ ->
             Just <| A.attribute "role" "alert"
 
```

Comment:
Får ikke til å legge kommentaren på linje 82, men denne bør være `Info` i stedet for `_`. På den måten, hvis man legger til flere varianter i `UserStatus` så vil du få en compile error for å ta stilling til hvordan den nye varianten bør håndteres.